core/src/main/java/fr/sncf/osrd/conflicts/SpacingResourceGenerator.kt

```diff
@@ -57,15 +63,18 @@ class SpacingRequirementAutomaton(
     // requirements that need to be returned on the next successful pathUpdate
     private val pendingRequirements = ArrayDeque<PendingSpacingRequirement>()
 
+    private var nextStopToProcess = 0
+    private var processedStops = ArrayDeque<ProcessedStop>()
+
```

Comment:
Should most likely be included in `clone()`

Reply:
Done in https://github.com/OpenRailAssociation/osrd/pull/7322/commits/1c9bce5bfc2c52375414a9b6f25297b7dba021e5